### ISSUE TITLE
Task generate_pv_ground_mounted fails

### ISSUE THREAD

**Issue by nesnoj** [openego/eGon-data#339]
I'm getting
```
Traceback (most recent call last):
  File "/home/local/RL-INSTITUT/jonathan.amme/virtualenvs/eGon-data/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 984, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/home/local/RL-INSTITUT/jonathan.amme/virtualenvs/eGon-data/lib/python3.8/site-packages/airflow/operators/python_operator.py", line 113, in execute
    return_value = self.execute_callable()
  File "/home/local/RL-INSTITUT/jonathan.amme/virtualenvs/eGon-data/lib/python3.8/site-packages/airflow/operators/python_operator.py", line 118, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/local/RL-INSTITUT/jonathan.amme/git-repos/openego/eGon-data/src/egon/data/processing/pv_ground_mounted.py", line 1132, in regio_of_pv_ground_mounted
    pv_rora_mv = pv_rora[pv_rora["voltage_level"] == 5]
  File "/home/local/RL-INSTITUT/jonathan.amme/virtualenvs/eGon-data/lib/python3.8/site-packages/geopandas/geodataframe.py", line 1299, in __getitem__
    result = super(GeoDataFrame, self).__getitem__(key)
  File "/home/local/RL-INSTITUT/jonathan.amme/virtualenvs/eGon-data/lib/python3.8/site-packages/pandas/core/frame.py", line 3455, in __getitem__
    indexer = self.columns.get_loc(key)
  File "/home/local/RL-INSTITUT/jonathan.amme/virtualenvs/eGon-data/lib/python3.8/site-packages/pandas/core/indexes/base.py", line 3363, in get_loc
    raise KeyError(key) from err
KeyError: 'voltage_level'
```
Can you reproduce that?

**Comment by ClaraBuettner** [comment]
I haven't seen this error before. Did you run the test mode or the normal mode?

**Comment by nesnoj** [comment]
In test mode. Cannot execute the normal mode as `osmtgmod_osm_import` is failing :( , cf. https://github.com/openego/eGon-data/issues/258#issuecomment-882432265

**Comment by ClaraBuettner** [comment]
@KathiEsterl have you ever seen this problem or have any idea why this can happen?

**Comment by KathiEsterl** [comment]
> @KathiEsterl have you ever seen this problem or have any idea why this can happen?

I haven't had this problem yet.
I just tried to reproduce it in different ways, but I couldn't. 

The error occurs in a line just needed for examination. I could delete this part of the code.
However, I am wondering where the problem is and this way it might not be solved, but hidden or postponed.

**Comment by nesnoj** [comment]
Thanks for checking @KathiEsterl !
I just started a clean run on your server and will get back to u with results..

**Comment by ClaraBuettner** [comment]
I can reproduce the error when pandas version 1.3.0 is installed. So this seems to be also related to recent changes in pandas.

**Comment by KathiEsterl** [comment]
> I can reproduce the error when pandas version 1.3.0 is installed. So this seems to be also related to recent changes in pandas.

I can confirm that. I get "key errors" in several tasks using pandas 1.3.0.

**Comment by nesnoj** [comment]
Temp. fix in 86464cb789a433488ee777539640275dfd8b789c

**Comment by nesnoj** [comment]
Outdated
